bokeh/models/sources.py

```diff
@@ -210,6 +210,18 @@ class RemoteSource(DataSource):
 class AjaxDataSource(RemoteSource):
     method = String('POST', help="http method - GET or POST")
 
+    overwrite = Bool(True, help="""
+    Overwrite or Update plot values from data source object
+    """)
+    max_size = Int(help="""
+    Maximum size that a plot could expand when data is retreive during
+    polling updates. Larger than that size, the data will shift to the right
```

Comment:
no need to reference plots here, this is purely about the max size of the data source buffer, regardless of any plots.